dist/rpm/openQA.spec

```diff
@@ -553,13 +554,13 @@ fi
 %service_del_postun openqa-continuous-update.timer
 
 %post local-db
-%service_add_post openqa-setup-db.service
+%service_add_post openqa_localdb_services
 
 %preun local-db
-%service_del_preun openqa-setup-db.service
+%service_del_preun openqa_localdb_services
 
 %postun local-db
-%service_del_postun openqa-setup-db.service
+%service_del_postun openqa_localdb_services
```

Comment:
Sorry if I'm missing something, but shouldn't these references be expanded to the variable defined earlier (i.e. something like `%{openqa_localdb_services}`)?